content/src/types.ts

```diff
@@ -90,3 +92,13 @@ export type StatusProbeResult = {
 export type IStatusCapableComponent = {
   getComponentStatus(): Promise<StatusProbeResult>
 }
+
+// TODO: Move this to catalyst-commons
+export function parseEntityType(strType: string): EntityType {
+  if (strType.endsWith('s')) {
+    strType = strType.slice(0, -1)
+  }
```

Comment:
I'd love to remove that `if` if possible

Reply:
Do you mean the logic of removing the s, or using an if instead of a regex?

Reply:
mm, I've only moved the method from another file. Doing that would result in a breaking change for the entire API